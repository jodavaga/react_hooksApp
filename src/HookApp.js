import React from 'react'

export const HookApp = () => {
    return (
        <div className="container">
            <h1>Hola Mundo</h1>
        </div>
    )
}
