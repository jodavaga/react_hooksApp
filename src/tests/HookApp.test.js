import React from 'react';
import {shallow} from 'enzyme';
import { HookApp } from '../HookApp';

describe('Tests on <HookApp />', () => {

    test('Should render <Hook />', () => {
        const wrapper = shallow( <HookApp /> );

        expect(wrapper).toMatchSnapshot();
    });
})