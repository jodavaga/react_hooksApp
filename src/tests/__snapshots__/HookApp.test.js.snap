// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <HookApp /> Should render <Hook /> 1`] = `
<div
  className="container"
>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
